pkg/pac/pac.go

```diff
@@ -97,8 +76,11 @@ func getNewSmeeURL() (string, error) {
 }
 
 func createSmeeDeployment(c *clients.Clients, namespace, smeeURL string) error {
+	/*
+		https://github.com/openshift-pipelines/pipelines-as-code
+		/pkg/cmd/tknpac/bootstrap/templates/gosmee.yaml
+	*/
```

Comment:
What's the value of this? We could at least use https://github.com/openshift-pipelines/pipelines-as-code/blob/main/pkg/cmd/tknpac/bootstrap/templates/gosmee.yaml so that it's easy to navigate to the file

Reply:
This is the reference template for gosmee.yaml